go-controller/pkg/config/config.go

```diff
@@ -642,6 +643,11 @@ var OvnNBFlags = []cli.Flag{
 			"Default value for this setting is empty which defaults to use local unix socket.",
 		Destination: &cliConfig.OvnNorth.CACert,
 	},
+	&cli.StringFlag{
+		Name:        "nb-cert-common-name",
+		Usage:       "Common Name of the certificate used for TLS server certificate verification",
```

Comment:
Specify what happens if this isn't provided (i.e. the CN needs to match the DNS)

Reply:
done.